databricks/koalas/indexes.py

```diff
@@ -679,14 +679,13 @@ def to_series(self, name: Union[str, Tuple[str, ...]] = None) -> Series:
         scol = self.spark.column
         if name is not None:
             scol = scol.alias(name_like_string(name))
+        elif len(kdf._internal.index_map) == 1:
+            name = self.name
         column_labels = (
             [(SPARK_DEFAULT_SERIES_NAME,)]
-            if len(kdf._internal.index_map) > 1
-            else [
-                (SPARK_DEFAULT_SERIES_NAME,) if name is None else name
-                for name in kdf._internal.index_names
-            ]
-        )
+            if len(kdf._internal.index_map) > 1 or name is None
+            else [name if isinstance(name, tuple) else (name,)]
+        )  # type: List[Tuple[str, ...]]
```

Comment:
Now seems working well, but this code still occurs an error in my local like the below.

`databricks/koalas/indexes.py:687: error: List item 0 has incompatible type "object"; expected "Tuple[str, ...]"`

Reply:
It doesn't happen in my local. Maybe your mypy version is old? Mine is `0.780` and GitHub Action's is `0.782`.

Reply:
> It doesn't happen in my local. Maybe your mypy version is old? Mine is `0.780` and GitHub Action's is `0.782`.

You're right, It was my mypy version problem. now It's working, thanks :)